rss/rss_parser.cpp

```diff
@@ -113,11 +113,12 @@ std::string rss_parser::__w3cdtf_to_rfc822(const std::string& w3cdtf) {
 		}
 	}
 
-	time_t t = mktime(&stm);
-	time_t x = time(NULL);
-	t += localtime(&x)->tm_gmtoff + offs;
+	//tm_isdst will force mktime to consider DST, like localtime(), but
+	//then the offset will be zeroed out, since that was manually added
```

Comment:
The old solution used the tm_gmtoff member, which apparently is not part of the standard, but part of BSD.  I had one solution that set that value manually, but that broke on the Apple builds.  The solution that's there now doesn't use that member variable.